### ISSUE TITLE
schema of number is float64

### ISSUE THREAD

**Issue by Tevic** [getkin/kin-openapi#280]
```
// Number
Min        *float64 `json:"minimum,omitempty" yaml:"minimum,omitempty"`
Max        *float64 `json:"maximum,omitempty" yaml:"maximum,omitempty"`
MultipleOf *float64 `json:"multipleOf,omitempty" yaml:"multipleOf,omitempty"`
```
I can see the schema of number is float64, however there will be precision problems between float64 and int64

**Comment by fenollp** [comment]
Indeed! I plan on moving schema validation to https://github.com/xeipuuv/gojsonschema which takes proper care of this using Reals. 
Now I'm wondering whether all float64 field should actually be reals. I don't think any JSON Schema Draft specifies a precision (and just assumes that of the js runtime). I know this was discussed and the above mentioned repo. So this will surely follow that discussion.
